drivers/dai/intel/alh/alh.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INTEL_DAI_DRIVER_ALH_H__
+#define __INTEL_DAI_DRIVER_ALH_H__
```

Comment:
```suggestion
#define __INTEL_DAI_DRIVER_ALH_H__
```
not following the pattern used on Zephyr.

Reply:
The majority of the drivers follow the `ZEPHYR_DRIVERS_<name>_H__` pattern. However, not every module and driver does it.